roles/logins/templates/limitedusers.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+_logger_tag="limitedusers"
+_uid="$(id -u "${PAM_USER}")"
+_user="${PAM_USER}"
+logger -t "${_logger_tag}" "[$0] runing under uid '$(id -u)' for user with uid '${_uid}'"
+
+# limit cpu resources
+{% if ansible_facts.processor_nproc >= 8 %}
+_cpuquota={{ ansible_facts.processor_nproc*10 }}  # cpus >= 8, each user gets 10% of total cpus
+{% elif ansible_facts.processor_nproc > 1 %} 
+_cpuquota={{ ansible_facts.processor_nproc*20 }} # cpus >=1 and cpus < 8, each user gets 20% of total cpus
+{% else %}
```

Comment:
This `else` will only be executed when the number of `ansible_facts.processor_nproc` is less then 1; Is that even possible?